src/utils/sampleFileTable.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable */
```

Comment:
Is there a specific reason for adding this as a seperate file rather than the fileTable

Reply:
I moved this file as it was in ICDC. I am not sure about that.